src/components/FlowRunFilteredList.vue

```diff
@@ -77,4 +77,10 @@
     selected.value = []
     subscriptions.refresh()
   }
-</script>
\ No newline at end of file
+</script>
+
+<style>
+.flow-run-filtered-list__search { @apply
+  min-w-[224px]
+}
```

Comment:
These are only semi related, but helps search maintain some width when the state input tries to take up a lot of horizontal space in the list headers.